### ISSUE TITLE
`metadata_to_file` is striping characters out of metadata filename

### ISSUE THREAD

**Issue by elehmer** [cal-adapt/climakitae#102]
When exporting to CSV is was noticing that the `g` was missing from the filename of the metadata text file created with the CSV file.

Looking at the code the `strip` function is used which will remove any occurance of ANY of the characters included.

https://github.com/cal-adapt/climakitae/blob/7f58d849d1199108170f2b202ab220d80360f161/climakitae/data_export.py#L269

So in this case `.csv.gz` removes all `.`, `c`, `s`, `v`, `g`, and `z` characters from the string.

**Comment by elehmer** [comment]
Here is the solution I thing we desire:

```
def rchop(s, suffix):
    if suffix and s.endswith(suffix):
        return s[:-len(suffix)]
    return s

somestring = 'this is some string rec'
rchop(somestring, ' rec')  # returns 'this is some string'
```

**Comment by elehmer** [comment]
Making PR...

**Comment by bethmints** [comment]
yes, this is an issue I noticed Friday and attempted to fix in PR #101 - looks like your fix is much more sophisticated, though. I'll be going over everything, thank you!

**Comment by elehmer** [comment]
> yes, this is an issue I noticed Friday and attempted to fix in PR #101 - looks like your fix is much more sophisticated, though. I'll be going over everything, thank you!

OMG I totally forgot about that change in #101 lol. That way works too. I did notice that the txt file ended up in a directory above where I was. You could just look over #103 and glean what I think is needed and then I can delete it.
